cob_hardware_config/cob4-1/config/head_self_collision.yaml

```diff
@@ -0,0 +1 @@
+
```

Comment:
yes, empty file is fine for now...it's only used for twist_control with CollisionAvoidance enabled...